openmdao/components/meta_model_unstructured_comp.py

```diff
@@ -205,6 +205,7 @@ def add_output(self, name, val=1.0, training_data=None, surrogate=None, **kwargs
         if surrogate:
             metadata['surrogate'] = surrogate
             metadata['default_surrogate'] = False
+            metadata['surrogate_name'] = surrogate.options._parent_name
```

Comment:
it seems like just setting this to `type(surrogate).__name__` would be more direct and more robust than than going through the secondary `options` object

Reply:
changed